doc/manuals/admin/install.md

```diff
@@ -42,7 +42,7 @@ Sometimes the above commands fails due to yum cache. In that case, run
 
 ### Using RPM file
 
-Download the package from the [FIWARE Files area](https://forge.fiware.org/frs/?group_id=7). Look for the "DATA-OrionContextBroker" entry.
+Download the package directly from the [FIWARE yum repository](https://nexus.lab.fiware.org/service/rest/repository/browse/el/7/x86_64/). You can choose release or nightly packages.
```

Comment:
How can I choose release or nightly packages?

Reply:
You will see 2 folders, nightly and release if try to use this link

Reply:
fixed in c6e45d153045d7d51e08a4dc3c732985c22fe5c9